OpenEdXMobile/src/test/java/org/edx/mobile/view/BaseFragmentActivityTest.java

```diff
@@ -119,7 +119,7 @@ protected void assertTitle(BaseFragmentActivity activity, CharSequence title) {
         ActionBar bar = activity.getSupportActionBar();
         assumeNotNull(bar);
         assumeNotNull(title);
-        Typeface type = ResourcesCompat.getFont(activity, R.font.open_sans_semi_bold);
+        Typeface type = ResourcesCompat.getFont(activity, R.font.inter_semi_bold);
```

Comment:
is there any way to use `attr` value instated of hardcoded font `R.font.inter_semi_bold`?